aws/resource_aws_cognito_user_pool_client.go

```diff
@@ -55,6 +55,12 @@ func resourceAwsCognitoUserPoolClient() *schema.Resource {
 						cognitoidentityprovider.ExplicitAuthFlowsTypeAdminNoSrpAuth,
 						cognitoidentityprovider.ExplicitAuthFlowsTypeCustomAuthFlowOnly,
 						cognitoidentityprovider.ExplicitAuthFlowsTypeUserPasswordAuth,
+						cognitoidentityprovider.ExplicitAuthFlowsTypeAllowAdminUserPasswordAuth,
+						cognitoidentityprovider.AuthFlowTypeAdminUserPasswordAuth,
```

Comment:
You won't need cognitoidentityprovider.AuthFlowTypeAdminUserPasswordAuth, just cognitoidentityprovider.ExplicitAuthFlowsTypeAllowAdminUserPasswordAuth

Reply:
done